data/json/recipes/recipe_food.json

```diff
@@ -2460,6 +2460,43 @@
       [ "nachos", 3 ]
     ]
   ]
+  },{
+  "type" : "recipe",
+  "result": "fresh_fries",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_VEGGI",
+  "skill_used": "cooking",
+  "difficulty": 3,
+  "time": 16000,
+  "reversible": false,
+  "autolearn": true,
+  "note": "the cooking oil isn't supposed to be expended since you can recycle it after frying the potatos",
```

Comment:
Ha, at least you know that these changes important for someone else and your PR was checked ;-)